docs/contributing.md

```diff
@@ -33,21 +33,94 @@ The next Community Call will be held on October 24th, 2024 at [5:00 PM CEST](htt
 Please use the [project's issue tracker](https://github.com/headlamp-k8s/headlamp/issues) for filing any bugs you find or features
 you think are useful.
 
+### Guidelines for Submitting an Issue
+
+When submitting an issue, follow these guidelines to help maintainers address it efficiently:
+
+**Search for Existing Issues:**
+
+- Use the issue tracker to see if your issue already exists. If it does, consider adding your input to the existing issue instead of opening a new one.
+- Use the template for issues depending on if the issue is a bug or feature
```

Comment:
![image](https://github.com/user-attachments/assets/d4137e2f-78dd-4259-9e73-39ecb8b5d65b)

Please consider this?

1. Using a section for "Search for Existing Issues"
2. Changing the second point to continue on from the first
 
```
#### Search for Existing Issues:

- Use the issue tracker to see if your issue already exists. If it does, consider adding your input to the existing issue instead of opening a new one.
- If the issue doesn't exist, please create a new issue using one of the templates available (bug, feature, security, or blank).
```

Reply:
forgot to post done